README.adoc

```diff
@@ -138,7 +138,9 @@ Flags:
   -n, --namespace string              Specify the namespace to operate in.
       --service string                Uses a Knative service as binding sink.
   -s  --sink string                   Sink expression to define the binding sink.
-      --property stringArray   Add a source property in the form of "<key>=<value>"
+      --property stringArray          Add a source property in the form of "<key>=<value>
+      --cloudEventsSpecVersion        Customize cloud events spec version provided to the binding sink.
+      --cloudEventsType               Customize cloud events type provided to the binding sink."
```

Comment:
Does the readme file need to be regenerated manually to reflect updated name?

Reply:
Otherwise looks good to me.

Reply:
it is a manual step 🙈  I have fixed it, thx